### PR TITLE
Skillonomy 120

### PR DESCRIPTION
**Pull request by RuslanSkiraRaccoongang**
**Description:** Put third party API IP into settings
[**Youtrack: skillonomy-120**] (https://youtrack.raccoongang.com/issue/SKILLONOMY-120)

**Testing instructions:**

**Reviewers:**
- [ ] @oleksandra-martyntseva 

**Merge checklist:**
- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation in source code updated
- [ ] All related Confluence documentation is updated (including deployment documentation)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: lms/djangoapps/student_profile/views.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def learner_performance(request, course_id=None, student_id=None):
         "api_endpoint": "profile_statistics",
         "key": getattr(settings, 'EDEOS_API_KEY'),
         "secret": getattr(settings, 'EDEOS_API_SECRET'),
-        "base_url": 'http://195.160.222.156/api/point/v1/'
+        "base_url": getattr(settings, 'EDEOS_API_STATISTICS')
```
Comment (oleksandra-martyntseva) [inline lms/djangoapps/student_profile/views.py]: please name the property EDEOS_API_STATISTICS_URL

File: lms/envs/aws.py [hunk 2]
```diff
@@ -1037,4 +1037,5 @@
 EDEOS_API_KEY = AUTH_TOKENS.get('EDEOS_API_KEY', '')
 EDEOS_API_SECRET = AUTH_TOKENS.get('EDEOS_API_SECRET', '')
 EDEOS_API_URL = AUTH_TOKENS.get('EDEOS_API_URL', '')
+EDEOS_API_STATISTICS = AUTH_TOKENS.get('EDEOS_API_STATISTICS', '')
```
Comment (oleksandra-martyntseva) [inline lms/envs/aws.py]: auth tokens are for secure data - like passwords. We can put urls to env tokens

Comment (RuslanSkiraRaccoongang) [reply lms/envs/aws.py]: fixed

File: lms/djangoapps/student_profile/views.py (hunk 1 above)
Comment (RuslanSkiraRaccoongang) [reply lms/djangoapps/student_profile/views.py]: fixed

**Pull request merged**